### DISCUSSION TITLE
yt --transcript not piping to fabric.

### DISCUSSION THREAD

**Discussion by blubomber** [danielmiessler/Fabric#529] [Q&A] [answered]
Hello, when i used the yt --transcript | fabric -sp extrac_wisdom, i can see the transcript on the screen but i get a message from ai that shows as follows. 

I'm sorry, but it seems like there's no input provided for me to process. Could you please provide the text content you'd like me to analyze?[1]-  Done                    yt --transcript https://www.youtube.com/watch?v=lQVcbY52_gY
[2]+  Done                    list=WL

So not sure where the issue is. This is running in WSL on a debian and Ubuntu distro. I can manually copy the transcript and do a 
paste | fabric -sp extract_wisdom and  get an output with no issues. And i am using ChatGPT. 

Thanks for any guidance.

**Comment by Tanner-Perham** [comment, marked as answer]
Hey @blubomber - I was having the same issue, I figured it out :)

**The Problem:** The link you are passing contains a '&' - this is making the command run in the background, so it basically will detatch the command from your terminal, runs it behind the scenes and then dumps the output to your current window.

**The Solution:** Wrap the whole yt command in brackets! This will stop it from detatching :)
`(yt --transcript <YOUR YOUTUBE LINK>) | fabric -sp extract_wisdom`

Good luck! Hope that helps!

**Reply by blubomber** [reply]
Tanner-Perham, that did the trick. Thank you!

**Reply by fearby** [reply]
That doesn't work for me  - just get zsh: no matches found: and then URL of YT video
